_faq/resource/scanners.md

```diff
@@ -21,7 +21,7 @@ For more information, see [scanner descriptions]({% link _docs/latest/configure/
 | Enabled APIs Scanner |
 | Firewall Rules Scanner |
 | Forwarding Rules Scanner |
-| Google Groups Scanner |
+| G Suite Scanner |
```

Comment:
Same issue as above, maybe Groups scanner makes more sense for now.

Reply:
Reverted.